packages/components/psammead-timestamp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-timestamp",
-  "version": "0.2.0",
+  "version": "0.3.1",
```

Comment:
I just noted that this other PR did not increment the version number https://github.com/bbc/psammead/pull/494#issuecomment-490192237 and so version 0.3.0 is merged on `latest` but not published. 

So can you please hold off on merging this PR until that fix has been addressed? Thanks.